src/pmpo_fortran.f90

```diff
@@ -487,6 +487,32 @@ subroutine polympo_getMeshVtxRotLat(mpMesh, nVertices, latitude) &
     type(c_ptr), value :: latitude
   end subroutine
   !---------------------------------------------------------------------------
+  !> @brief set the polympo mesh elements/cells center
+  !> @param mpmesh(in/out) MPMesh object
+  !> @param nElements(in) length of array in, use for assertion 
+  !> @param x/y/zArray(in) the 1D arrays of element centers coords 
+  !---------------------------------------------------------------------------
+  subroutine polympo_setMeshElmCenter(mpMesh, nElements, xArray, yArray, zArray) &
```

Comment:
use 'nCells' instead of 'nElements' like other routines which take nCells as argument

Reply:
Done